src/users.rs

```diff
@@ -232,4 +259,43 @@ mod tests {
             "User was not deleted"
         );
     }
+
+    #[test]
+    fn test_load_users_from_yaml() {
+        // Setup
+        env::set_var("DB_FILE", "test.db");
+        env::set_var("USER_CONFIG", "test_users.yaml");
+        // Create the yaml File
+        let file_user = FileUser {
+            name: "file_user".to_string(),
+            password: "123456789".to_string(),
+        };
+        let mut file_users = Vec::new();
+        file_users.push(file_user);
+        let file = FileUsers { users: file_users };
+        let yaml_string = serde_yaml::to_string(&file).expect("Failed Creating Yaml String");
+        match File::create("test_users.yaml") {
+            Ok(mut file) => file
+                .write_all(yaml_string.as_bytes())
+                .expect("Unable to write data"),
+            Err(_) => assert!(false, "Failed Creating file"),
+        }
+        // Test Loading the yaml file into db
+        assert!(
+            User::load_users_from_file().is_ok(),
```

Comment:
I'm tempted to say change this to just:

```
User::load_users_from_file().unwrap();
```
Or `expect` if you want the explicit message. But admittedly using asset does make it clearer what we're testing.